packages/apps/halo-app/src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -10,6 +10,7 @@ export type ProgressBarProps = {
   className?: string;
 };
 
+// TODO(burdon): Move to react-ui?
```

Comment:
`react-ui` has `Status` which does the same thing, so this component should be deprecated.

Reply:
@zhenyasav